command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
+rucio account update
+rucio account ban add
+rucio account ban remove
+rucio account attribute list
+rucio account attribute add
+rucio account attribute remove
+rucio account limit list
+rucio account limit add
+rucio account limit remove
+rucio account identity list
+rucio account identity add
+rucio account identity remove
+```
+
+# Config 
+```
+rucio config list
+rucio config add
+rucio config remove
+```
+
+# DID 
+```
+rucio did list
+rucio did show
+rucio did remove
+rucio did touch
+rucio did parent
+rucio did attach
+rucio did detach
+rucio did upload
+rucio did download
+rucio did content list
+rucio did content history
+rucio did content close
+rucio did content open
+rucio did metadata list
+rucio did metadata add
+rucio did metadata remove
+rucio did container add
+rucio did dataset add
+```
+
+# Replica 
+```
+rucio replica list
+rucio replica dataset
+rucio replica pfn
+rucio replica tombstone
+rucio replica state suspicious
+rucio replica state quarantine
+rucio replica state bad
+rucio replica state temp-unavailable
+```
+
+# RSE
+```
+rucio rse list
+rucio rse show
+rucio rse remove
+rucio rse add
+rucio rse update
+rucio rse attribute list
+rucio rse attribute add
+rucio rse attribute remove
+rucio rse distance list
+rucio rse distance add
+rucio rse distance remove
+rucio rse distance set
+rucio rse protocol add
+rucio rse protocol remove
+rucio rse limit add
+rucio rse limit remove
+rucio rse qos list
+rucio rse qos add
+rucio rse qos remove
+
+```
+
+# Rule 
+```
+rucio rule add
+rucio rule remove
+rucio rule show
+rucio rule history
+rucio rule update
+rucio rule list
+```
+
+# Scope 
+```
+rucio scope [list]
+rucio scope add
+```
+
+# Subscription
+```
+rucio subscription list
+rucio subscription update
+rucio subscription add
+rucio subscription touch
+
+```
+
+# Lifetime-Exception 
+```
+rucio lifetime-exception [add]
+```
```

Comment:
This somewhat reaffirms my concern for a default. If it’s not the same everywhere, then it becomes a source of inconsistency. I would rather the user had to be explicit.

`lifetime-exception` needs some additional thinking. It’s due to the fact that lifetimes can appears elsewhere (e.g. replication rules), but this is about the Lifetime Model.

Likewise, we need to consider the subcommands `list`, `show`, `update`, and `remove`.

Reply:
I'm not married to the defaults, I thought it would just be a nice QOL feature for those who don't have tab completing setup. 

Lifetime was also a huge question mark for me, I'm not sure how it's generally used, so I will defer to you.